workspaces/desktop/server/package.json

```diff
@@ -7,9 +7,9 @@
   "main": "index.js",
   "dependencies": {
     "@novnc/novnc": "1.4.0",
-    "command-line-args": "^6.0.0",
+    "command-line-args": "^6.0.1",
```

Comment:
Minor concern: there's no proper changelog. It seems the only change is in tests for Node v23, so it should be fine.